<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>514</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="hotColdWaterGroupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="hotWaterGroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="hotWaterLabel">
            <property name="text">
             <string>HotWater      </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="hotWaterValuesGroupBox">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="2" column="1">
              <widget class="QLabel" name="hotWaterPreviousLabel">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="hotWaterCurrentLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="hotWaterTariffLabel">
               <property name="text">
                <string>Tariff</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="hotWaterTariffLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>176.83</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="hotWaterCurrentLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="hotWaterPreviousLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="coldWaterGroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="coldWaterLabel">
            <property name="text">
             <string>ColdWater    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="coldWaterValuesGroupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="1" column="2">
              <widget class="QLineEdit" name="coldWaterTariffLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>45.88</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="coldWaterCurrentLabel">
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="coldWaterPreviousLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="coldWaterTariffLabel">
               <property name="text">
                <string>Tariff</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="coldWaterPreviousLabel">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="coldWaterCurrentLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="wateSinkGroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="waterSinkLable">
            <property name="text">
             <string>WaterSink     </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="waterSinkValuesGroupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="1" column="1">
              <widget class="QLabel" name="waterSinkTariffLabel">
               <property name="text">
                <string>Tariff</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="waterSinkTariffLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>35.5</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="electricityGroupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="t1GroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="t1Label">
            <property name="text">
             <string>Electricity T1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="t1ValuesGroupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="1" column="1">
              <widget class="QLabel" name="t1TariffLabel">
               <property name="text">
                <string>Tariff</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="t1PreviousLabel">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="t1CurrentLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="t1PreviousLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="t1CurrentLabel">
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="t1TariffLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="t2GroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="t2Label">
            <property name="text">
             <string>Electricity T2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="t2ValuesGroupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="3" column="2">
              <widget class="QLineEdit" name="t2CurrentLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="t2PreviousLabel">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="t2CurrentLabel">
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="t2PreviousLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="t2TariffLabel">
               <property name="text">
                <string>Tariff</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="t2TariffLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="t3GroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="t3Lable">
            <property name="text">
             <string>Electricity T3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="t3ValuesGroupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <item row="2" column="1">
              <widget class="QLabel" name="t3PreviousLabel">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="t3TariffLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="t3TariffLabel">
               <property name="text">
                <string>Tariff</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="t3CurrentLabel">
               <property name="text">
                <string>Current</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="t3PreviousLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="t3CurrentLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="calculateGroupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="resultLabel">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
